import-chrome/src/import-chrome.cpp

```diff
@@ -158,14 +177,23 @@ int main( int argc, char** argv )
         {
             for ( auto& kv : v["args"].items() )
             {
-                zoneText += kv.key() + ": " + kv.value().dump() + "\n";
+                const auto val = kv.value();
+                const std::string s =
+                    val.is_string() ?
+                    val.get<std::string>() : val.dump();
+                zoneText += kv.key() + ": " + s + "\n";
             }
         }
 
+        uint64_t pid = 0;
+        if ( v.contains( "pid" ) ) {
+            pid = v["pid"].get<uint64_t>();
+        }
```

Comment:
Why is PID extracted here, but TIDs are inside the `if` bodies?

Reply:
good question, I mostly followed the existing structure of the code. I can fix that.

Reply:
It certainly looks like `getTid()` could have just one json parametr and extract tid and pid itself.